tracing-subscriber/src/filter/env/directive.rs

```diff
@@ -476,6 +476,12 @@ impl Dynamics {
             None
         }
     }
+
+    pub(crate) fn has_value_filters(&self) -> bool {
+        self.directives
+            .iter()
+            .any(|d| !d.fields.iter().any(|f| f.value.is_some()))
```

Comment:
AAAGHHHHH i thought i rebased this
```suggestion
            .any(|d| d.fields.iter().any(|f| f.value.is_some()))
```